src/land/ArcanistLandEngine.php

```diff
@@ -86,6 +87,15 @@
     return $this;
   }
 
+  final public function getRevision() {
```

Comment:
You can move these methods  ant `private $revision;` to `ArcanistSubmitQueueEngine` class. As they are used in mentioned engine only. So we'll leave this file untouched.